app/views/devise/sessions/new.html.erb

```diff
@@ -1,37 +1,31 @@
-<% content_for :title, "Sign in" %>
-<% content_for :thin_form, true %>
+<% content_for :title, "Sign in to GOV.UK" %>
 
-<div class="page-title">
-  <h1>Sign in</h1>
-</div>
-
-<!--[if lte IE 7]>
-  <div class="alert alert-warning add-bottom-margin">
-    <h4>Your browser is not supported</h4>
-    <p>Pages might not work as expected. You should <a href="https://www.gov.uk/help/browsers" class="link-inherit">upgrade your browser</a> – you might need to ask your IT administrator to do this for you.</p>
-  </div>
-<![endif]-->
-<%= form_for(resource, as: resource_name, url: session_path(resource_name), html: { class: 'panel panel-default', autocomplete: 'on' }) do |f| %>
-  <div class="panel-body">
-    <div class="form-group">
-      <%= f.label :email %>
-      <%= f.email_field :email, class: 'form-control input-lg', tabindex: 1, autofocus: 'autofocus' %>
```

Comment:
A reminder for us to update the input component to allow `tabindex` and `autofocus`. I'll try and do it later today.